Lib/test/test_typing.py

```diff
@@ -2870,6 +2870,13 @@ def foo(a: 'Callable[..., T]'):
         self.assertEqual(get_type_hints(foo, globals(), locals()),
                          {'a': Callable[..., T]})
 
+    def test_special_forms_forward(self):
+
+        class C:
+            a: Annotated['ClassVar[int]', (3, 5)] = 4
```

Comment:
Let's test Final too just for sure, and also test failing cases like `List["Final[int]"]`.

Reply:
good suggestion